From 1e419fcbaf2bcb17407525965d5e4ca7bc35a719 Mon Sep 17 00:00:00 2001
From: Michael Forney <mforney@mforney.org>
Date: Sun, 12 Feb 2017 23:01:21 -0800
Subject: [PATCH] Fix build with HAVE_GL==0

video/out/opengl/hwdec.h includes video/out/opengl/common.h, which tries
to include opengl headers.
---
 options/options.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/options/options.c b/options/options.c
index b711156c1..0bb36ad3f 100644
--- a/options/options.c
+++ b/options/options.c
@@ -40,7 +40,6 @@
 #include "stream/stream.h"
 #include "video/csputils.h"
 #include "video/hwdec.h"
-#include "video/out/opengl/hwdec.h"
 #include "video/image_writer.h"
 #include "sub/osd.h"
 #include "audio/filter/af.h"
@@ -53,6 +52,10 @@
 #include "video/out/drm_common.h"
 #endif
 
+#if HAVE_GL
+#include "video/out/opengl/hwdec.h"
+#endif
+
 extern const char mp_help_text[];
 
 static void print_version(struct mp_log *log)
-- 
2.11.1

